### DISCUSSION TITLE
Want to run and optimize AlGaN based material for UVC LED using OpenModelica? any help and advise.

### DISCUSSION THREAD

**Discussion by mazwan-ai** [OpenModelica/OpenModelica#10623] [Q&A]

**Comment by AnHeuermann** [comment]
Hey @mazwan-ai. I moved your question from issues to discussions. The issues are more about bugs and problems with the software. Here you can reach more people for general questions on the usage.
Please add a bit more information on what you want to do and maybe someone from the community can help you.

**Comment by phannebohm** [comment]
You can find some information on optimization tasks with OpenModelica [here](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/optimization.html).
